modules/cooler/dump/main.nf

```diff
@@ -28,11 +29,12 @@ process COOLER_DUMP {
     script:
     def software = getSoftwareName(task.process)
     def prefix   = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
+    def postfix  = resolution?"::$resolution":""
     """
     cooler dump \\
         $options.args \\
         -o ${prefix}.bedpe \\
-        $cool
+        $cool$postfix
```

Comment:
Where is the resolution parameter passed on to the cooler command? Am I missing something?